src/ontology/pato-edit.obo

```diff
@@ -20612,6 +20613,54 @@ synonym: "skin redness" EXACT []
 is_a: PATO:0000051 ! morphology
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: PATO:0040050
+name: diandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have two stamens." [http://orcid.org/0000-0002-2908-3327] {comment="https://orcid.org/0000-0001-8815-0078"}
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040051
+name: monandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have only one stamen." [http://orcid.org/0000-0002-2908-3327, https://orcid.org/0000-0001-8815-0078]
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040052
```

Comment:
when I open the OWL file, the ID for this term is PATO_0070037

Reply:
![image](https://user-images.githubusercontent.com/6722114/122827201-ea23aa80-d298-11eb-9640-0eb0789835c1.png)

Reply:
I dont know, something seems really wrong here.. Maybe lets do this together in our face2face on Thursday. The only thing that can be wrong is the subclassOf axioms - the terms should be correct.